### DISCUSSION TITLE
About the  Overview of ML Compilation

### DISCUSSION THREAD

**Discussion by LRY89757** [mlc-ai/mlc-en#4] [General]
Will some resources about first ML Compilation Lecture be released today ?
<br>
![image](https://user-images.githubusercontent.com/77330637/174262545-73a9263d-15c0-45a8-a30a-835c3d3fc8e0.png)

**Comment by Hzfengsy** [comment]
It's the date of US timezone :)

**Reply by LRY89757** [reply]
All right :），thanks! Hope there will be a Chinese version in the future : ).

**Reply by Hzfengsy** [reply]
It's already there. Please see https://mlc.ai/summer22-zh/ and https://mlc.ai/zh

**Reply by LRY89757** [reply]
Wow! Many thanks to you. I have only seen the English version before.
